purpler/bot.py

```diff
@@ -189,18 +189,31 @@ def on_pubmsg(self, c, e):
 
         self._handle_command(c, e)
 
-        results = TRANSCLUDER.finditer(message)
+        results = EMBEDDER.finditer(message)
         if results:
             for result in results:
-                guid = result.group(1)
-                self.log.debug('saw guid %s', guid)
+                mode = result.group(1)
+                if mode == "l":
+                    link_only = True
+                elif mode == "t":
+                    link_only = False
+                else:
```

Comment:
is this even possible? group(1) can only be t or l according to the regex?

Reply:
Uh, good point... :man_facepalming: Will rewrite.